locations/spiders/aramark_uniform_services_us.py

```diff
@@ -65,4 +66,7 @@ def parse(self, response):
             elif "Cleanroom Services" in item["name"].title():
                 item["brand"] = "Aramark Cleanroom Services"
                 item["name"] = item["brand"] + " " + item["city"]
+
+            apply_category(Categories.SHOP_CLOTHES, item)
+            apply_category({"rental": "clothes"}, item)
```

Comment:
I think categories should be applied in the `if/elif` statements above? If it's `Uniform Services` or `Cleanroom Services`, then `shop=laundry` (with other tags to denote it's a private laundry that customers can't just walk into) may be most suitable? For `Cleanroom Services`, I see that they advertise themselves as supplying and maintaining reusable cleanroom garments for businesses, hence I think these locations are primarily where they clean garments, rather than a shopfront where customers can walk in to buy garments, or where they warehouse new garments for distribution to customers.

Reply:
Looking more deeply into it they do offer laundry services as well as uniform rentals. Okay so uniform services will be clothe rentals and cleanroom will be laundry. I can get behind that.

Also Looks like Aramack changed its name to Vestis last month.

Reply:
Thanks for investigating. I see now why it's difficult to decide between `rental:clothes` and `shop:laundry` as the difference is seemingly only who owns the garments, and these type of industrial laundry services would probably allow for both (for example, a hotel that has their own branded linen, or another hotel that just uses plain linen mixed with other hotels).

We can rename this spider as needed.

Reply:
Does that mean you are okay with what I added below? Having Cleanroom Services being laundry and the general being rental clothes?

Looking into it more. 
> today announced that Vestis™ will be the name of the new company to be created through the expected spin-off of its uniform services business.
I don't think we need to change the spider name. Seems vestis will be its own thing. But the names of the shops that are being scraped all have Vestis in it.

Reply:
I think we should delete:
```
apply_category(Categories.SHOP_CLOTHES, item)
```

(as I think this category is more for places you can walk into to buy a bunch of goods and then walk out with those goods)

And perhaps both cleanroom and uniform services get the same categories:
```
apply_category(Categories.SHOP_LAUNDRY, item)
apply_yes_no("laundry_service", item, True)
apply_category({"rental": "clothes"}, item)
apply_category({"access": "private"}, item)
```

Which would mean it's a private laundry service that either accepts privately owned garments, or garments previously rented from the business?

Generally `brand` should match the name the location is marketed as, which sounds like `Verstis` in this case. I agree that the spider name can remain the same as current, but I think we should override `brand` and `brand_wikidata` for uniform services.

Reply:
As `Vestis` has been spun off as a new company and new brand, a new Wikidata item is needed as there are now two separate companies and thus a need for two Wikidata items.

Reply:
Have created a new wikidata item for Vesits Corporation (https://www.wikidata.org/wiki/Q122947676)